### PR TITLE
2776 lesmis emergency preparedness dashboard

### PR DESCRIPTION
**Pull request by EMcQ-BES**
### Changes:

- Add Emergency in Education subDashboard processing to lesmis
- Add migration to add dashboards

---

### Screenshots:
Attached to ticket

### CODE REVIEW

File: packages/database/src/migrations/20210705111800-LESMISAddEmergencyInEducationDashboard-modifies-data.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  for (const { name, code, sort_order } of DASHBOARD_LIST) {
+    await insertObject(db, 'dashboard', {
+      id: generateId(),
+      code,
+      name,
+      root_entity_code: 'LA',
+      sort_order,
+    });
+    await addItemToDashboard(db, {
+      code: 'no_data_at_level',
```
Comment (EMcQ-BES) [inline packages/database/src/migrations/20210705111800-LESMISAddEmergencyInEducationDashboard-modifies-data.js:81]: This is just an easy way to make lesmis display the `There are no reports available for this dashboard` message

Comment (edmofro) [reply packages/database/src/migrations/20210705111800-LESMISAddEmergencyInEducationDashboard-modifies-data.js:81]: Clever!

**Review by edmofro — CHANGES_REQUESTED** [review]
I think you still need to do the school level dashboard (just the tab "Emergency/Disaster Affected" within the School Profile)

**Comment by EMcQ-BES** [comment]
> I think you still need to do the school level dashboard (just the tab "Emergency/Disaster Affected" within the School Profile)

Good catch, I should have made a note, that was added in this PR https://github.com/beyondessential/tupaia/pull/2754 (Along side adjusting the tab orders)

**Comment by edmofro** [comment]
Oohh, nice

**Review by edmofro — APPROVED** [review]

**Pull request merged**